web/src/main/java/com/futurewei/alcor/web/entity/dataplane/refactor/NetworkConfiguration.java

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2019 The Alcor Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.web.entity.dataplane.refactor;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.schema.Common.OperationType;
+import com.futurewei.alcor.schema.Common.ResourceType;
+import com.futurewei.alcor.web.entity.dataplane.InternalPortEntity;
+import com.futurewei.alcor.web.entity.dataplane.InternalSubnetEntity;
+import com.futurewei.alcor.web.entity.dataplane.NeighborEntry;
+import com.futurewei.alcor.web.entity.dataplane.NeighborInfo;
+import com.futurewei.alcor.web.entity.route.InternalRouterInfo;
+import com.futurewei.alcor.web.entity.securitygroup.SecurityGroup;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class NetworkConfiguration {
+
+  private ResourceType rsType;
```

Comment:
In v2 NetworkConfiguration, could we get rid of rsType and OpType, which are supposed to be DPM internal and not part of the contract.

Reply:
If we remove rsType and OpType, DPM should add more rest APIS to identity which resource and operation should update.

Reply:
rsType and OpType are resource-level fields, not on the message level. That being said, there will be OpType inside each top-level resource. For example, router_internal always has field already, and we will add to more top-level resources.

Reply:
Well, I think we can keep these two fields, and add OpType in top-level resources in the future.